resources/views/debug.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use Yiisoft\Assets\AssetManager;
+use Yiisoft\View\WebView;
+use Yiisoft\Yii\Debug\Viewer\Asset\DevPanelAsset;
+
+/**
+ * @var WebView $this
+ * @var AssetManager $assetManager
+ */
+
+$assetManager->register(DevPanelAsset::class);
+
+$this->addCssFiles($assetManager->getCssFiles());
+$this->addCssStrings($assetManager->getCssStrings());
+$this->addJsFiles($assetManager->getJsFiles());
+$this->addJsStrings($assetManager->getJsStrings());
+$this->addJsVars($assetManager->getJsVars());
+
+$this->beginPage();
+?>
+    <!doctype html>
+    <html lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width,initial-scale=1"/>
+        <meta name="description" content="Aggregator for Yii3 specific frontends"/>
```

Comment:
What's meant by "Aggregator" here? I guess the description should say something else.